code/modules/mob/living/living.dm

```diff
@@ -130,6 +127,10 @@ default behaviour is:
 					return
 			// VOREStation Edit - End
 
+			if(a_intent == I_HELP || src.restrained())
```

Comment:
These lines of code changed should have //VOREStation edit on it if they're moved.

Reply:
They shouldn't, all that was done here was that he edit was moved.

Reply:
Ah, very well. Was just wondering why it showed the line being modified.

Reply:
Mm, very well. Was wondering why github referenced this.